test/integration/scheduler/framework_test.go

```diff
@@ -599,6 +637,165 @@ func TestUnreservePlugin(t *testing.T) {
 	}
 }
 
+// TestBindPlugin tests invocation of bind plugins.
```

Comment:
It seems this test is flaky:

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/79294/pull-kubernetes-integration/1142430172373323779/

Reply:
Met the same problem in https://github.com/kubernetes/kubernetes/pull/78513/files